src/main/scala/com.snowplowanalytics/iglu.schemaddl/migrations/FlatSchema.scala

```diff
@@ -134,6 +142,18 @@ object FlatSchema {
       (schema.canBeNull, getName(pointer))
     }
 
+  def buildOrderedSubSchemas(source: SchemaList): OrderedSubSchemas =
```

Comment:
Can we add a description, for me it looks like one of the most important functions.